File: Python/Kode Program/Manusia.py
class Manusia:
    def __init__(self, NIK, nama, umur, jenisKelamin):
        self.NIK = NIK
        self.nama = nama
        self.umur = umur
        self.jenisKelamin = jenisKelamin
    
    def getNIK(self):
        return self.NIK
    
    def setNIK(self, NIK):
        self.NIK = NIK
    
    def getNama(self):
        return self.nama
    
    def setNama(self, nama):
        self.nama = nama

    def getUmur(self):
        return self.umur
    
    def setUmur(self, umur):
        self.umur = umur
    
    def getJenisKelamin(self):
        return self.jenisKelamin
    
    def setJenisKelamin(self, jenisKelamin):
        self.jenisKelamin = jenisKelamin

File: CPP/Kode Program/Manusia.cpp
#ifndef MANUSIA_H
#define MANUSIA_H

#include <iostream>
#include <vector>

using namespace std;

class Manusia{
    //Atribut Class Manusia
    protected:
        int NIK;
        string nama;
        string umur;
        string jenisKelamin;

    //Constructor
    public:
        Manusia(int NIK, string nama, string umur, string jenisKelamin){
            this->NIK = NIK;
            this->nama = nama;
            this->umur = umur;
            this->jenisKelamin = jenisKelamin;
        }

        //Getter dan Setter
        int getNIK(){
            return this->NIK;
        }

        void getNIK(int NIK){
            this->NIK = NIK;
        }

        string getNama(){
            return this->nama;
        }

        void setNama(string nama){
            this->nama = nama;
        }

        string getUmur(){
            return this->umur;
        }

        void setUmur(string umur){
            this->umur = umur;
        }

        string getJenisKelamin(){
            return this->jenisKelamin;
        }

        void setJenisKelamin(string jenisKelamin){
            this->jenisKelamin = jenisKelamin;
        }

        ~Manusia(){
        }
};

#endif